docs_en/getting-started.md

```
@@ -0,0 +1,121 @@
+    
+    [Learn More →](wiki/index.md){ .more-link }
+
+-   :fontawesome-solid-user:{ .twemoji } 
+    
+    **User Guide**
+    
+    ---
+    
+    Detailed usage instructions and best practices
+    
+    [Coming Soon](){ .more-link }
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)